mcda/utils/utils_for_main.py

```diff
@@ -426,17 +464,25 @@ def save_df(df: pd.DataFrame, folder_path: str, filename: str):
     :param filename: str
     :return: None
     """
+    custom_output_path = os.environ.get('PROMCDA_OUTPUT_PATH')  # check if an environmental variable is set
+    output_directory_path = custom_output_path if custom_output_path else DEFAULT_OUTPUT_DIRECTORY_PATH
```

Comment:
Same comment as for default_input_dir_path.
Also, want to inform user if you selected default path or the one specified by the user? May be add some logs of print on console?

Reply:
The user is informed where the input file is read and where the output files are saved (see comment above). However, the user is not specifically informed if the directories are the default ones. Could it be cluttering the log info?